.github/workflows/test262.yml

```diff
@@ -57,9 +57,8 @@ jobs:
           cd boa
           comment="$(./target/release/boa_tester compare ../gh-pages/test262/refs/heads/main/latest.json ../results/test262/pull/latest.json -m)"
           comment="${comment//'%'/'%25'}"
-          comment="${comment//$'\n'/'%0A'}"
           comment="${comment//$'\r'/'%0D'}"
-          echo "::set-output name=comment::$comment"
+          echo "comment=$comment" >> $GITHUB_OUTPUT
```

Comment:
Since the result output contains newlines, we need to use delimiters for it.

https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions#multiline-strings

Reply:
Hi! 
Thanks for your time.  I pushed out the changes that use EOF, not sure if I got it right. I'm pretty green when it comes to CI/CD stuff, but I really like the project and would like to add my contribution!

Thank you!